src/test/resources/minimizer/TestInput3.java.expected

```diff
@@ -0,0 +1,14 @@
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class TestInput3Minimized {
+
+    // Test simplification of right hand side values
+    @Test
+    public void test1() throws Throwable {
+        int d = 0;
```

Comment:
The test cases TestInput1 and TestInput3 are a bit unfortunate.
They do not help a programmer detect the error.
For example, suppose the input was
```
  int x = sqrt(4);
  assert x == 2;
```
but someone optimized sqrt incorrectly so that now it produces 1.999, so the assertion failed.
The minimizer would change this to
```
  int x = 0;
  assert x == 2;
```
which fails for the same reason but has nothing to do with the original error.
This suggests that minimization might not be desirable, and perhaps should not be enabled by default.
Or maybe just that our purely dynamic approach to minimization is undesirable, and I was wrong to suggest it.
Or maybe this kind of result doesn't occur very often and so it isn't a concern in practice (though just in case, we would want Randoop to output both the original and the minimized test suite).